Makefile

```diff
@@ -33,5 +41,21 @@ generate-ci-config:
 
 # Generate an aggregated knative yaml file with replaced image references
 generate-release:
-	./openshift/release/generate-release.sh $(RELEASE)
+	@test $(RELEASE_VERSION) || { echo "You need to set the RELEASE_VERSION on the command line i.e: make RELEASE_VERSION=0.4.0"; exit ;1;}
```

Comment:
nit: this works although a better way may to define a target like `pre-req` and use `ifndef` e.g.

```Makefile
# default
all: validate
  @echo works

validate:
  ifndef REL_VER
    $(error REL_VER must be set)
  endif

novalidation:
  @echo works without validation

```

Reply:
sounds good, i didn't go that way because i feel it's a bit of unnecsary complexity but may be more makefileness elegant